W-14244982 Adding operations and outbound variables (#308)
mulesoft/docs-rpa: rpa-builder/6.5/modules/ROOT/pages/toolbox-variable-handling-string-operations.adoc

= String Operations

> The String Operations action step enables you to modify strings by using a variety of operations, including find and replace and trimming.

Reviewer: ```suggestion
The String Operations action step enables you to modify strings by using several operations, including find and replace and trimming.

```
